web/rootfs/defaults/settings-config.js

```diff
@@ -554,7 +554,11 @@ config.e2eping.maxMessagePerSecond = {{ .Env.E2EPING_MAX_MESSAGE_PER_SECOND }};
 // Settings for the Excalidraw whiteboard integration.
 config.whiteboard = {
     enabled: {{ $WHITEBOARD_ENABLED }},
-    collabServerBaseUrl: '{{ $WHITEBOARD_COLLAB_SERVER_PUBLIC_URL }}'
+{{ if .Env.WHITEBOARD_COLLAB_SERVER_PUBLIC_URL -}}
+    config.whiteboard.collabServerBaseUrl = '{{ $WHITEBOARD_COLLAB_SERVER_PUBLIC_URL }}';
```

Comment:
This syntax is incorrect, needs to be likle the other URL.

Reply:
Which other URL? Isn't this the same syntax used for Etherpad?

https://github.com/jitsi/docker-jitsi-meet/blob/09bbfaf01cc400d85d21006ccb33ac40da21366b/web/rootfs/defaults/settings-config.js#L143-L147

Reply:
Not the same, here we are declaring an object literal.

Reply:
Sorry, i'm not following you: i just tried to run this locally and the syntax in this file seem to be correct, there was an unrelated issue in another file (`meet.conf`) but with that fixed the whiteboard is running fine on my end.

Was this something that was requested and then addressed in the original PR and i am now reverting it accidentally in this new PR?

Reply:
The case that doesn't work is the previous one, not the one you are adding.

Looks perhaps like a merge error. Just make sure the collab... property is assigned the same way not with config.foo.bar =. While that works in other scenarios, it doesn't in an object literal as is the case here.

Reply:
Understood, this is due to things being moved around in https://github.com/jitsi/docker-jitsi-meet/commit/1dbe5cda68038403b417caf641ce476f0fa7d571#diff-8fc682fc9b342e54e76c53511199952847429c949bca05b186ae167f883f18d0L480, should be fixed now.